specification/schemas/CarrierContract.json

```diff
@@ -12,7 +12,6 @@
       "properties": {
         "attributes": {
           "type": "object",
-          "additionalProperties": false,
```

Comment:
Why is this removed?

Reply:
With post carrier contract, we want to specify that you can send credentials. So that's done with an `"allOf"` in the post path. But `"allOf"` is not possible when `"additionalProperties": false` is present.